sessionctx/variable/tidb_vars.go

```diff
@@ -355,6 +355,9 @@ const (
 	// TiDBEnablePointGetCache is used to control whether to enable the point get cache for special scenario.
 	TiDBEnablePointGetCache = "tidb_enable_point_get_cache"
 
+	// TiDBEnableAlterPartitionPlacement is used to control whether to enable alter table partition.
+	TiDBEnableAlterPartitionPlacement = "tidb_enable_alter_partition_placement"
```

Comment:
There will be many similar features in the future, such as alter table placement, alter index placement. How about just add only one switch for all instead of one switch for each feature?

Reply:
yes, we can.  I will change the name later.
we can judge this variable for every alter placement type in the entrance of DDL.